README.md

```diff
@@ -3,11 +3,11 @@ Localytics for PhoneGap/Cordova
 
 ## Version
 
-This version of the PhoneGap/Cordova SDK wraps v4.5.1 of the Localytics Android and v4.4.1 of the Localytics iOS SDKs.
+This version of the PhoneGap/Cordova SDK wraps v5.1.0 of the Localytics Android and v5.1.0 of the Localytics iOS SDKs.
 
 ## Supported Versions
 
-The PhoneGap/Cordova SDK was tested on Cordova v7.1.0 with Android v6.3.0 and iOS v4.5.4.
+The PhoneGap/Cordova SDK was tested on Cordova v8.0.0 with Android v6.3.0 and iOS v4.5.4.
```

Comment:
The comment is wrong. The Android version has been changed in the Cordova release 8.0.0. The version is 7.0.0. I am not sure why we are identifying the Android and iOS version. We should just list out the Cordova version.